content/docs/studio/user-guide/account-management.md

```diff
@@ -82,10 +82,140 @@ In this section, you can,
 
 ## Cloud credentials
 
-Here, you can see all the credentials you have added for your cloud / data
+Here, you can see all the credentials you have added for your cloud and data
 remotes. You can also add new credentials as well as edit or delete existing
-credentials. The credentials that you add here will become available to use with
-your projects in Iterative Studio.
+credentials. These credentials useful in two cases:
+
+- To
+  [fetch project data from data remotes](/doc/studio/user-guide/projects-and-experiments/configure-a-project#data-remotes--cloud-storage-credentials)
+- To
+  [create cloud resources for running experiments](/doc/studio/user-guide/projects-and-experiments/run-experiments#cloud-experiments)
+
+To add credentials, click on your user icon in the top right corner and go to
+your [Profile]. Navigate to `Cloud Credentials` and click `Add credentials`.
+Select the cloud provider (note that
+[cloud experiments](/doc/studio/user-guide/projects-and-experiments/run-experiments#cloud-experiments)
+currently support AWS and GCP). Depending on the provider, you will be asked for
+more details.
+
+![](https://static.iterative.ai/img/studio/s3_remote_settings_v2.png)
+
+The credentials must have the required permissions. For
+[cloud experiments](/doc/studio/user-guide/projects-and-experiments/run-experiments#cloud-experiments),
+the following permissions are needed:
+
+<details>
+
```

Comment:
Not a blocker, but we should follow up to make this more user-friendly. We can include the credentials video from @dacbd and mention like he does that an easy way to get started is to give full access to ec2 and s3. We could also provide an example policy that grants all the permissions below so people can copy and paste it.

I'm less familiar with GCP/Azure, but we should be able to do something similar.

Reply:
Create [an issue](https://github.com/iterative/dvc.org/issues/4613). I'll try to work on it tomorrow.